lib/Varien/Data/Form/Abstract.php

```diff
@@ -139,7 +140,13 @@ public function addField($elementId, $type, $config, $after = false)
         } else {
             $className = 'Varien_Data_Form_Element_' . ucfirst(strtolower($type));
         }
-        $element = new $className($config);
+
+        if (class_exists($className)) {
+            $element = new $className($config);
+        } else {
+            $className = 'Varien_Data_Form_Element_Note';
+            $element = new $className($config);
+        }
```

Comment:
is this also neecssary for this PR? cause I normally would prefer the error instead

Reply:
if not add these lines, we would meet errors in the customer/view and order/view page at the backend.
because sometimes newsletter or giftmessage added attribute to the entity. if disabled the module later, the attribute related entity view page would get error. 

This error is not obviously before, and if we need error, we can use another way to notify backend operator. but not stop them using the system with blank page.